lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1730,4 +1730,57 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                       binder.op, resultType, input);
                   return success();
                 });
+
+  patterns.onOp(
+      "Hardmax", 13, [](OpBinder Binder, ConversionPatternRewriter &Rewriter) {
+        // def hardmax(tensor, dim=-1):
+        //   zeros = torch.zeros_like(tensor, dtype=torch.int64)
+        //   maximums = torch.argmax(tensor, dim=dim, keepdim=True)
+        //   return zeros.scatter(dim, maximums, value=1)
+
+        Torch::ValueTensorType ResultType;
```

Comment:
Variable names should be `camelCase`. So change `ResultType` to `resultType`. Same for other variables below.

Reply:
updated.